readme.md

```diff
@@ -22,43 +22,43 @@ The plugin streamlines the entire configuration process, and removes the need fo
 - Automatic webhook creation in a Bitbucket Server repo when a Jenkins job is saved.
 - Quick selection of Bitbucket Server projects and repos for a Jenkins job through a dropdown.
 - The ability to automatically send build statuses to Bitbucket Server.
-- Cloning from Bitbucket Server Smart Mirrors with no need to modify the clone URL.
+- Cloning from Bitbucket Server [Smart Mirrors](https://confluence.atlassian.com/display/BitbucketServer/smart+mirroring) with no need to modify the clone URL.
 
 ## Requirements
 
 - Jenkins 2.190.1+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 5.6 to 7.3 are supported, but they aren't recommended as creating an Application Link with one of these versions doesn't provide any additional functionality.
+Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+ as when you also set up an Application Link you will have access to all plugin features. 
```

Comment:
@mhenschke-atl in response to that [feedback](https://issues.jenkins-ci.org/browse/JENKINS-64022)  I've updated this line. What do you think?

Reply:
I think this looks good. The points of clarification seem to be:
- If you have 7.4 and above, there are extra features we recommend
- Those features need an applink
- A subset are still available 7.3 and below, or without the applink.
I think this covers that well. Cheers Cam!